ERCS/erc-7921.md

```diff
@@ -0,0 +1,135 @@
+---
+eip: 7921
```

Comment:
```suggestion
eip: 7829
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.